### DISCUSSION TITLE
Shorthand for RowDefinitions and ColumnDefinitions

### DISCUSSION THREAD

**Discussion by Arlodotexe** [CommunityToolkit/Labs-Windows#483] [Ideas]
Creating Grid `RowDefinitions` and `ColumnDefinitions` in XAML can be quite verbose:

```xaml
<Grid>
    <Grid.RowDefinitions>
        <RowDefinition Height="300"/>
        <RowDefinition Height="*"/>
        <RowDefinition Height="2*"/>
    </Grid.RowDefinitions>

    <Button />
    <Button Grid.Row="1"/>
    <Button Grid.Row="2"/>
</Grid>
```

Rather than type that out every time, we could create a shorthand that reduces this code to something like:
```xaml
<Grid wct:RowDefinitions="300,*,2">
    <Button />
    <Button Grid.Row="1"/>
    <Button Grid.Row="2"/>
</Grid>
```

**Comment by michael-hawker** [comment]
This is already a feature in WinUI 3: https://github.com/microsoft/microsoft-ui-xaml/issues/673 - Though not sure where the VS bug is for the tooling being broken, maybe a new one needs to be filed?

Don't feel like we should polyfill just for UWP for this one, as convenient as it can be.
